pkg/messaging/brokers/brokers_nats.go

```diff
@@ -0,0 +1,37 @@
+//go:build nats
+// +build nats
+
```

Comment:
Consider adding `init` function to log for which broker the binary is built (https://go.dev/doc/effective_go#init).